pr_check.sh

```diff
@@ -4,11 +4,14 @@
 # Export vars for helper scripts to use
 # --------------------------------------------
 # name of app-sre "application" folder this component lives in; needs to match for quay
-export COMPONENT="insights-inventory" 
-export APP_NAME=`node -e 'console.log(require("./package.json").insights.appname)'`
+export COMPONENT="insights-inventory"
+export APP_NAME=$(node -e 'console.log(require("./package.json").insights.appname)')
 export IMAGE="quay.io/cloudservices/$COMPONENT-frontend"
-export WORKSPACE=${WORKSPACE:-$APP_ROOT}  # if running in jenkins, use the build's workspace
+export IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```

Comment:
Do I understand it correctly that `IMAGE` and `IMAGE_TAG` can be omitted?

Reply:
The docker image doesn't use them, but we still need them for the quay push. They just don't need to be passed into `docker run`